### PR TITLE
[EN-1241] Expand AggregateException to provide meaningful info about error

### PR DESCRIPTION
**Pull request by ilya-shablovsky**
Made Bridge AggregateException be meaningful by expanding it
https://github.com/lunalabsltd/integration/pull/1064

### CODE REVIEW

**Review by vladimir-tikhonov — APPROVED** [review]

**Review by vladimir-tikhonov — APPROVED** [review]

**Review by zhemchugovstepan — APPROVED** [review]

**Review by Tokatak — APPROVED** [review]

**Review by MakeevAlexey — CHANGES_REQUESTED** [review]
For now all bridge errors are shown as one projDiag message let's make them separate.
Also try to cut away as much unnecessary info as possible

File: Compiler/Builder/Program.cs [hunk 1]
```diff
@@ -59,7 +59,7 @@ private static int Main(string[] args)
                 }
                 else
                 {
-                    logger.Error(string.Format("Bridge.NET Compiler error: {0}", ex.ToString()));
+                    logger.Error(string.Format("Bridge.NET Compiler error: {0}", ex.Message));
```
Comment (MakeevAlexey) [inline Compiler/Builder/Program.cs:62]:
```suggestion
                    logger.Error(string.Format("Bridge.NET Compiler error: \n{0}", ex.Message));
```

File: Compiler/Translator/Translator/Translator.InspectAssembly.cs [hunk 2]
```diff
@@ -409,10 +409,10 @@ private void BuildSyntaxTreeForFile(int index, ref string[] rewriten)
                 var errors = new List<string>();
                 foreach (var error in parser.Errors)
                 {
-                    errors.Add(fileName + ":" + error.Region.BeginLine + "," + error.Region.BeginColumn + ": " + error.Message);
+                    errors.Add("[ErrorMessage]" + fileName + ":" + error.Region.BeginLine + "," + error.Region.BeginColumn + ": " + error.Message);
```
Comment (MakeevAlexey) [inline Compiler/Translator/Translator/Translator.InspectAssembly.cs:412]: please use $"", and put a space after "[ErrorMessage] "

File: Compiler/Translator/Translator/Translator.cs [hunk 3]
```diff
@@ -159,8 +159,21 @@ public void Translate()
...
+            catch (AggregateException ex)
+            {
+                var exceptions = ex.InnerExceptions;
+                foreach (var exception in exceptions)
+                {
+                    var error = $"{exception.Message}\n";
+                    logger.Error(error);
+                }
 
+                throw new Exception("Compilation Error! For details see errors above.", ex);
```
Comment (MakeevAlexey) [inline Compiler/Translator/Translator/Translator.cs:175]: code styles

**Review by MakeevAlexey — CHANGES_REQUESTED** [review]

File: Compiler/Translator/Translator/Translator.cs (hunk 3 above)
Comment (ilya-shablovsky) [reply Compiler/Translator/Translator/Translator.cs:175]: @MakeevAlexey means spaces inside paranthesis.
And Bridge code style is differs from Luna one's, so, there are no parenthesis spaces across the project.
We discussed it directly and decided to leave as is.

File: Compiler/Translator/Translator/Translator.InspectAssembly.cs (hunk 2 above)
Comment (ilya-shablovsky) [reply Compiler/Translator/Translator/Translator.InspectAssembly.cs]: done

File: Compiler/Builder/Program.cs (hunk 1 above)
Comment (ilya-shablovsky) [reply Compiler/Builder/Program.cs]: done

**Review by MakeevAlexey — APPROVED** [review]

**Pull request merged**